includes/Embed_Block.php

```diff
@@ -169,7 +169,8 @@ protected function render_block_html( array $attributes ) {
 		$title        = (string) $attributes['title'];
 		$poster       = ! empty( $attributes['poster'] ) ? esc_url( $attributes['poster'] ) : '';
 		$align        = sprintf( 'align%s', $attributes['align'] );
-		$player_style = sprintf( 'width: %dpx; height: %dpx', absint( $attributes['width'] ), absint( $attributes['height'] ) );
+		$margin       = ( 'center' === $attributes['align'] ) ? 'auto' : '0';
```

Comment:
Should we perhaps consider creating a stylesheet file with the following?

```css
.wp-block-web-stories-embed.aligncenter {
  margin-left: auto;
  margin-right: auto;
}
```

Reply:
I did think about it, but is there any point in loading another stylesheet on page just for 3 lines of css?